package.json

```diff
@@ -6,6 +6,7 @@
   "license": "MIT",
   "main": "dist/react-powerplug.cjs.js",
   "module": "dist/react-powerplug.esm.js",
+  "types": "types/index.d.ts",
```

Comment:
Is it mandatory rule to place d.ts in types folder or can we keep it in `src` like .js.flow (src is published on npm)?

Reply:
This is d.ts requirement, as lots of other, there is no way to pass (their) linting without keeping this structure.
For the typescript itself - it does not matter.

Reply:
Then add "types" to "files" array field.

Reply:
done yet again.